canonical-web-and-design/vanilla-framework
# `p-tab-buttons` pattern

Co-authored-by: Scott Mason Nash <scott.mason.nash@canonical.com>

## Done

Added the tab buttons pattern

Fixes #3887 & #3897

## QA

- Open [demo](https://vanilla-framework-3921.demos.haus/docs/examples/patterns/tab-buttons/default)
- Check that the pattern looks and behaves as expected
- Review updated documentation:
  - https://vanilla-framework-3921.demos.haus/docs/patterns/tab-buttons

### Check if PR is ready for release

If this PR contains Vanilla SCSS code changes, it should contain the following changes to make sure it's ready for the release:

  - `Feature 🎁`, `Breaking Change 💣`, `Bug 🐛`, `Documentation 📝`, `Maintenance 🔨`.
  - if CSS class names are not changed it can be bugfix relesase (x.x.**X**)
  - if CSS class names are changed/added/removed it should be minor version (x.**X**.0)
  - see the [wiki for more details](https://github.com/canonical-web-and-design/vanilla-framework/wiki/Release-process#pre-release-tasks)
